lib/matplotlib/rcsetup.py

```diff
@@ -312,10 +312,12 @@ def validate_color(s):
     if isinstance(s, str):
         if s.lower() == 'none':
             return 'none'
-        if len(s) == 6 or len(s) == 8:
-            stmp = '#' + s
-            if is_color_like(stmp):
-                return stmp
+        # Support "rrggbb"/"rggbbaa" without leading '#' because these would be
```

Comment:
rggbbaa -> rrggbbaa

it's just in comments, but...

Reply:
thanks for noting that, fixed.